can-stream_test.js

```diff
@@ -91,17 +91,16 @@ test('Compute streams do not bind to the compute unless activated', function ()
 		},
 		toCompute: function(makeStream, context) {}
 	};
+
 	var canStreaming = canStream(canStreamInterface);
 	var stream = canStreaming.toStream(c);
 
-	QUnit.equal(c.computeInstance.__bindEvents, undefined);
```

Comment:
@justinbmeyer I did not see this being carried over to `can-event-queue/map/map` so I'm guessing checking for `.bound` isthe right thing to do here.

Reply:
looks good